src/App.js

```diff
@@ -141,17 +140,41 @@ function stateOfWorld() {
 }
 
 function easy() {
-  var mode = "easy";
-  localStorage.setItem("mode", mode);
+  localStorage.setItem("difficultyValue", 15);
+  window.location.reload();
+}
+
+function medium() {
+  localStorage.setItem("difficultyValue", 30);
   window.location.reload();
 }
 
 function hard() {
-  var mode = "hard";
-  localStorage.setItem("mode", mode);
+  localStorage.setItem("difficultyValue", 46);
```

Comment:
if these are hardcoded (which is fine), why do we have the global variable?

Reply:
It make things easier for now.